InterfaceFactory/CommHandler4Modbus/src/test/java/communicator/rest/http/client/ApacheRestServiceClientTest.java

```diff
@@ -0,0 +1,110 @@
+package communicator.rest.http.client;
+
+import java.util.Map;
+import java.util.HashMap;
+
+import org.apache.hc.core5.http.NameValuePair;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+class ApacheRestServiceClientTest {
+
+    @Test
+	void testFormDataFromBodySuccessNoParams() {
+
+        final String body = "";
+        final Iterable<NameValuePair> params = ApacheRestServiceClient.getFormParameters(body);
```

Comment:
This does not compile (anymore). There is no static method `ApacheRestServiceClient.getFormParameters(body)`

Reply:
I fixed it in my last commit.

The only thing left would be to adapt the _SGrRestAPIDeviceTest_ (I was not sure how to capture the body string)